specs/address_book_list/02_user_interface.rst

```diff
@@ -0,0 +1,38 @@
+=====================
```

Comment:
Which screens are missing?

- Detail view
- "Add" view
- Empty state of the list (When there are no entries)

Anything else?

I'll create a ticket in the design backlog and link it to this epic so the design team can pick it up :)

Reply:
Also, how would an entry look like in the send flow?